subsys/audio_module/Kconfig

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Audio"
+
+#----------------------------------------------------------------------------#
+menu "Modules"
+
+config AUDIO_MODULE
+	bool "Enable the audio module"
```

Comment:
Given that the library depends on DATA_FIFO, should this option have a `depends on` Kconfig dependency that sets DATA_FIFO to `y`?

Reply:
Me or Erik can assist with that.